supplementary/alica_ros_turtlesim/ros1/CMakeLists.txt

```diff
@@ -62,7 +64,7 @@ target_link_libraries(${PROJECT_NAME}
                         constraintsolver
 )
 
-add_subdirectory(libalica-turtlesim)
+add_subdirectory(../libalica-turtlesim ../libalica-turtlesim)
```

Comment:
Adding twice seems wrong

Reply:
Because the "subdirectory" is not an actual subdirectory of ros1 we have to provide a relative build path for it. This allows us to build alica_ros_turtlesim without it being a package

Reply:
This seems even more wrong now, we have same line twice, with two instances of same directory in each

Reply:
Basically, I understand specifying binary directory (repeat of libalica-turtlesim) but repeat of hte entire line definitely seems wrong

Reply:
what do you mean by repeat the entire line? With the first we select the libalia-turtlesim folder which is another folder on the same level. With the second we specify where the binary files will be stored. For example,

`add_subdirectory(../libalica-turtlesim ../libalica-turtlesim)`

Will store the files in `build/libalica-turtlesim/`

`add_subdirectory(../libalica-turtlesim libalica-turtlesim)`

Will store the files in `build/alica_ros_turtlesim/libalica-turtlesim`, or for ros 2 in `build/alica_ros2_turtlesim/libalica-turtlesim`